pro/rac.rst

```diff
@@ -0,0 +1,79 @@
+===================================
+Netmaker Remote Access Client (RAC)
+===================================
+
+The Netmaker Remote Access Client, RAC for short is a GUI tool for easily getting access to a Netmaker network.
+RAC is mostly suited for offsite machines that need access to a Netmaker network. It supports Windows, Mac and Linux (mobile support coming soon).
+
+
+***********************
+Download/Installation
+***********************
+
+You can download the latest version of RAC from the Netmaker file server:
+
+.. code-block::
+
+  https://fileserver.netmaker.org/releases/download/<version>
+
+Search for remote client and download the appropriate version for your operating system.
+
+For Linux (Debian/Ubuntu), you can also use the following command to download the latest version:
+
+.. code-block:: 
+
+   curl -sL 'https://apt.netmaker.org/remote-client/gpg.key' | sudo tee /etc/apt/trusted.gpg.d/remote-client.asc
+   curl -sL 'https://apt.netmaker.org/remote-client/debian.deb.txt' | sudo tee /etc/apt/sources.list.d/remote-client.list
+   sudo apt update
+   sudo apt search remote-client  # to see available versions
+   sudo apt install remote-client
+
+
+******************
+Quick Start
+******************
+
+To use RAC, you will need to have a Netmaker server running and have a user account on that server. You will also need to have a client gateway set up on the server that you have access to.
+
+NOTE: RAC currently works with non-admin users
```

Comment:
this is not a limitation, the idea is to only work with non-admin users

Reply:
addressed